setup.py

```diff
@@ -5,10 +5,17 @@
 from setuptools.command.test import test as TestCommand
 from setuptools.command.build_ext import build_ext as BuildExtCommand
 
-# ORDER MATTERS
-# Import this after setuptools or it will fail
-from Cython.Build import cythonize  # noqa: I100
-import Cython.Distutils
+try:
+    # ORDER MATTERS
+    # Import this after setuptools or it will fail
+    from Cython.Build import cythonize  # noqa: I100
+    import Cython.Distutils
+except ImportError:
+    raise ImportError(
+        "Failed to import Cython modules. This can happen under versions of pip older than 18 that don't "
+        "support installing build requirements during setup. If you're using pip, make sure it's a "
+        "version >=18."
```

Comment:
```suggestion
        "Failed to import Cython modules. This can happen under versions of pip older than 18 that don't "
        "support installing build requirements during setup. If you're using pip, make sure it's a "
        "version >=18."
```


.... or manually install Cython?

I think we want to keep this message as-is and not suggest an alternative (only suggest upgrading pip), but wanted to call it out if we wanted to discuss it.

Reply:
Should we link to our public quickstart docs as well?

https://ddtrace.readthedocs.io/en/stable/installation_quickstart.html

My thinking here is if any install/build requirements change, then the error message can stay partially up to date.